Fix/3855 oss2.2 (#3857)
influxdata/docs-v2: content/influxdb/v2.2/write-data/best-practices/resolve-high-cardinality.md

    parent: write-best-practices
---

> If reads and writes to InfluxDB have started to slow down, high [series cardinality](/influxdb/v2.1/reference/glossary/#series-cardinality) (too many series) may be causing memory issues. {{% cloud-only %}}Cardinality can also cause writes to fail if it exceeds your [plan’s adjustable service quota](/influxdb/cloud/account-management/limits/).{{% /cloud-only %}}

Reviewer: ```suggestion
If reads and writes to InfluxDB have started to slow down, high [series cardinality](/influxdb/v2.1/reference/glossary/#series-cardinality) might be causing memory issues. {{% cloud-only %}}Cardinality can also cause writes to fail if it exceeds your [plan’s adjustable service quota](/influxdb/cloud/account-management/limits/).{{% /cloud-only %}}
```

Author: Thanks. Leaving as-is b/c I think Kelly might have originally suggested it.
